src/main/java/frc/robot/subsystems/IntakeShooter.java

```diff
@@ -132,19 +139,54 @@ public Command retractIntakeMotorsCommand() {
   }
 
   public Command shoot() {
-    /*
-     * i1.set(0.35);
-     * i2.set(0.35);
-     * s1.set(1);
-     * s2.set(1);
-     */
     System.out.println("SHOOTING");
     return Commands.sequence(this.retractIntakeMotorsCommand(), new WaitCommand(Constants.INTAKE_RETRACT_TIME),
-        this.stopIntakeMotorsCommand(), 
+        this.stopIntakeMotorsCommand(),
         this.startShooterMotorsCommand(),
-        new WaitCommand(Constants.SHOOTER_SPIN_UP_TIME), 
+        new WaitCommand(Constants.SHOOTER_SPIN_UP_TIME),
         this.intakeMotorShootCommand(),
-        new WaitCommand(Constants.SHOOTER_SHOOT_TIME), 
+        new WaitCommand(Constants.SHOOTER_SHOOT_TIME),
         this.stopShooterMotorsCommand());
   }
+
+  // TODO Check all under this comment
+  public Command limitSwitchOverride(boolean override) {
```

Comment:
Do we need this function if all it does is return the value from another function?

Reply:
Fair enough, I was just trying to follow somthing similar to the system above but I realize that is now obselete